scripts/tsc-with-whitelist.sh

```diff
@@ -63,6 +63,7 @@ web-common/src/runtime-client/http-request-queue/HttpRequestQueue.ts: error TS25
 web-common/src/runtime-client/http-request-queue/HttpRequestQueueTypes.ts: error TS18048
 web-common/src/runtime-client/http-request-queue/HttpRequestQueueTypes.ts: error TS2322
 web-common/src/lib/time/config.ts: error TS2322
+web-common/src/features/file-explorer/new-files.ts: error TS2741
```

Comment:
Is this a challenging type error? If easy, it'd be good to fix rather than expand this list.

Reply:
I'm not sure if I made any changes that caused this in this PR

Reply:
I think it's from this addition to the `ResourceKind` enum:

```
RefreshTrigger = "rill.runtime.v1.RefreshTrigger",
```